axelrod/tests/strategies/test_player.py

```diff
@@ -27,6 +27,52 @@ def randomize(*args):
     return random.choice([C, D])
 
 
+class ParameterisedPlayer(Player):
+    """A simple Player class for testing init parameters"""
+
+    def __init__(self, parameter1=None):
+        super().__init__()
+
+
+class TestPlayerParameters(unittest.TestCase):
+    """A class to test the initialisation parameters of Players"""
+
+    def test_player(self):
+        """Tests for Players with no init parameters"""
+
+        # Test that the init_kwargs dict exists and is empty
+        player = Player()
+        assert player.init_kwargs == {}
```

Comment:
Could this be: `self.assertEquals(player.init_kwards, {})`.

I believe that `assert` would not run in the same fashion: a failure would give an error as opposed to a test failure (which are reported differently by unittest).